README.md

```diff
@@ -1,23 +1,51 @@
-# Beetbox - Drupal development on Vagrant [![Circle CI](https://circleci.com/gh/drupalmel/beetbox.svg?style=svg)](https://circleci.com/gh/drupalmel/beetbox)
-
-## Overview
+# Beetbox - Drupal development on Vagrant
```

Comment:
I figured that change was still pending, but we can go ahead with that change prior to beta 1. Really only wanted to push the approved changes although I did sneak the **Features** change in which is still pending.